src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,24 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li className="is-active" data-cy="Tab" key={tab.id}>
```

Comment:
The `className` attribute is set to `is-active` for all tabs, which means all tabs will appear active. You should conditionally apply this class only if `tab.id === activeTabId`.